submissions/SerhiiYakovenko/tiny-js-world/index.js

```diff
@@ -0,0 +1,112 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const boi = {
+   species: 'human',
+   name: 'Dadia Fedor',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'Я ничей. Я сам по себе мальчик. Свой собственный.',
+   friends: []
+}
+
+const mama = {
+   species: 'human',
+   name: 'Rimma',
+   gender: 'female',
+   legs: 2,
+   hands: 2,
+   saying: 'Это не техника дошла, а я сама сюда дошла, на лыжах!',
+   friends: []
+}
+
+const papa = {
+   species: 'human',
+   name: 'Dmitriy',
+   gender: 'male',
+   legs: 2,
+   hands: 2,
+   saying: 'Здравствуйте! Угадайте, кто я?',
+   friends: []
+}
+
+const kot = {
+   species: 'cat',
+   name: 'Matroskin',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Не правильно ты, дядя Фёдор, бутерброд ешь...',
+   friends: []
+}
+
+const pes = {
+   species: 'dog',
+   name: 'Sharik',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Попрошу внимания! Сделайте, пожалуйста, умные лица!',
+   friends: []
+}
+
+/* Define cat-woman?
+   There is not cat-woman in prostokvashino.
+   But where is pochtalion peckin?
+   Haha, vot ge on:
+*/
+const pochtalion = {
+   ...papa
+}
+pochtalion.name = "Pochtalion Pechkin"
```

Comment:
You are mixing two types of quotes - single quotes and double quotes. Usually in JS we use single quotes, but most importantly whatever you choose, use it consistently.